xiaogpt/bot/edgegpt_bot.py

```diff
@@ -0,0 +1,47 @@
+from EdgeGPT import Chatbot, ConversationStyle
+from rich import print
+
+from xiaogpt.utils import split_sentences
+
+
+class EdgeGPTBot:
+    def __init__(self, cookie_file, conversation_style="creative", proxy=None):
```

Comment:
It seems `proxy` is not used, you don't have to pass it in as an (useless) argument.